packages/api-electron/main.js

```diff
@@ -1,7 +1,9 @@
 const { app } = require('electron');
 const ssfElectron = require('./index.js');
 const fs = require('fs');
+const path = require('path');
 const program = require('commander');
+const download = require('download');
```

Comment:
Are you using the `download` package as a way to get a promise-based interface for http requests?

Perhaps it is better to look for a more generic approach that can turn any node-callback interface into a promise? e.g. [util.promisify](http://2ality.com/2017/05/util-promisify.html) (or polyfill).

Reply:
Actually it was more a way to easily fetch a file without having to worry about http/https and reading a stream etc.

Can we use Node v8 functions? I don't think we currently specify a minimum version of Node, and the Node LTS is 6.11.4.